src/rust/src/expr/string.rs

```diff
@@ -59,13 +59,8 @@ impl PlRExpr {
             .into())
     }
 
-    fn str_zfill(&self, alignment: &PlRExpr) -> Result<Self> {
-        Ok(self
-            .inner
-            .clone()
-            .str()
-            .zfill(alignment.inner.clone())
-            .into())
+    fn str_zfill(&self, length: &PlRExpr) -> Result<Self> {
+        Ok(self.inner.clone().str().zfill(length.inner.clone()).into())
```

Comment:
Arg names in Rust didn't change for starts_with() and ends_with()

Reply:
In that case, the error message is unclear if an argument type mismatch occurs on the Rust side (which may not happen here).
Could you update the Rust side argument names to match the R side?

Reply:
Done